### ISSUE TITLE
Period Column + Top2

### ISSUE THREAD

**Issue by Piwington** [ONSdigital/es-aggregation-sg#31]
During Testing It Was Remembered That Imputation Apply Only Contain One Period. Therefore All These Wranglers And Methods That Use The Period Column Are Redundant As There Will Never Be More Than One.

**Comment by Piwington** [comment]
When we remove the period_column it may be best to rejig Top2 due to being hard to follow and possibly overly complicated. Mike and Myself have pondered a possibly simpler solution.

**Comment by KrisRogos** [comment]
PR #35 approve, this solved this issue.
